### PR TITLE
Type fixes

### PR DESCRIPTION
**Pull request by OleanjiKingCode**
# Fixed the 'as unknown' used in types

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/1284

### CODE REVIEW

File: src/components/wallet/use-hiiiq-balance.ts [hunk 1]
```diff
@@ -56,17 +56,14 @@ export const useHiIQBalance = (address: string | undefined | null) => {
   useEffect(() => {
     const getBalance = async () => {
       const hiiqBalance = Number(
-        formatUnits(balanceOf as unknown as bigint, 18),
+        formatUnits(BigInt(balanceOf?.toString() ?? 0), 18),
       )
-      const lockBalance = locked as unknown as any
+      const lockBalance: any = locked
```
Comment (Emmanueldmlr) [inline src/components/wallet/use-hiiiq-balance.ts:61]: any reason why you used any?

File: src/components/wallet/use-hiiiq-balance.ts [hunk 2]
```diff
@@ -56,17 +56,13 @@ export const useHiIQBalance = (address: string | undefined | null) => {
   useEffect(() => {
     const getBalance = async () => {
       const hiiqBalance = Number(
-        formatUnits(balanceOf as unknown as bigint, 18),
+        formatUnits(BigInt(balanceOf?.toString() ?? 0), 18),
```
Comment (Royal-lobster) [inline src/components/wallet/use-hiiiq-balance.ts:59]: Incase balanceOf is bigint or null, we can do type checking so it reads less complex

```ts
const fetchedBalance = typeof balanceOf === "bigint" ? balanceOf : 0
const hiiqBalance = Number(formatUnits(fetchedBalance), 18)
```

**Review by Royal-lobster — APPROVED** [review]
Looks good !, left a minor suggestion.

File: src/components/wallet/use-hiiiq-balance.ts (hunk 2 above)
Comment (OleanjiKingCode) [reply src/components/wallet/use-hiiiq-balance.ts:59]: its not of type bigint, its Result | never but i will make it less complex

Comment (Royal-lobster) [reply src/components/wallet/use-hiiiq-balance.ts]: Ohh, then cool !

**Review by Royal-lobster — APPROVED** [review]

**Pull request merged**